tests/benchmarks/pystone3.py

```diff
@@ -57,56 +57,60 @@
                 making the benchmark an integer benchmark again.
 
 """
+from time import time
```

Comment:
I wasnt sure whether to explicitly exclude this file.  It is authored by this project, but these changes will make it harder to correlate with the pystone.py which it is based on.

Reply:
Such changes are unlikely at this point, plus what one can always do, is to format the upstream. But in principle you are right.

Reply:
Actually, now that I saw that `pystone.py` is still excluded, I remembered that I actually did that Python3 port at some point myself, I doubt anybody else ever cared to port it, so you can lift that branch too, doesn't really matter anymore.

Reply:
> ... can lift that branch too,

Do you mean remove it from the exclusions, and reformat it, and push an update to this PR?
If so, any other changes I should make?